JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -481,15 +488,16 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
     cell.delegate = collectionView;
 
     if (!isMediaMessage) {
-        cell.textView.text = [messageItem text];
-
-        if ([UIDevice jsq_isCurrentDeviceBeforeiOS8]) {
-            //  workaround for iOS 7 textView data detectors bug
+        if ([messageItem conformsToProtocol:@protocol(JSQMessageAttributedData)])
+        {
+            id <JSQMessageAttributedData> attributedMessageItem =  (id <JSQMessageAttributedData>) messageItem;
+            cell.textView.attributedText = [attributedMessageItem attributedText];
+        }
+        else
+        {
             cell.textView.text = nil;
```

Comment:
Is there a reason for not alternating between .text and .attributedText? Why are we always creating attributed text here?